### DISCUSSION TITLE
Settings changes won't save

### DISCUSSION THREAD

**Discussion by jamminjames** [LiveHelperChat/livehelperchat#2024] [Q&A] [answered]
When I try to change a setting and save it, there is no indication it has not saved. However, if I go and come back, the setting is as it was before. How can I fix this?

**Comment by remdex** [comment]
Very detailed report I have to say, no URL nothing - where you change something and where it's not saved. Reproduce on demo and give EXACT steps to reproduce. https://livehelperchat.com/demo-12c.html

**Reply by jamminjames** [reply, marked as answer]
Sorry for the lack of detail. I think it was a permissions issue, I changed file permissions recursively for all the \livehelperchat folders to chmod 664, and that seems to have done it.
